package.json

```diff
@@ -15,6 +15,7 @@
     "http-status-code": "2.1.0",
     "http-status-codes": "2.1.4",
     "immutable": "4.0.0-rc.12",
+    "lodash": "4.17.21",
```

Comment:
You already have `lodash.truncate` perhaps you want to only import the needed function instead of the whole package.

Reply:
lodash._ is also used so I keep this dependecy unchanged